solver/src/main/java/org/chocosolver/solver/expression/continuous/arithmetic/UnCArExpression.java

```diff
@@ -48,7 +48,7 @@
     /**
      * The expression this expression relies on
      */
-    private CArExpression e;
+    CArExpression e;
```

Comment:
I believe this modification is not needed anymore if `CstCArExpression` disappears.